indexing-service/src/test/java/io/druid/indexing/overlord/TaskLifecycleTest.java

```diff
@@ -109,8 +125,25 @@
 import java.util.concurrent.Executor;
 import java.util.concurrent.TimeUnit;
 
+@RunWith(Parameterized.class)
 public class TaskLifecycleTest
 {
+
+  @Parameterized.Parameters(name = "taskStorageType={0}")
+  public static Collection<String[]> constructFeed()
+  {
+    return Arrays.asList(new String[][]{{"HeapMemoryTaskStorage"}, {"MetadataTaskStorage"}});
```

Comment:
```constructFeed()``` is called only once during the lifetime of the test suites. The instances of TaskStorage passed here won't be refreshed for each test since it's constructed once. So the construction of TaskStorage has to be in ```setUp()``` in order for it to be refreshed for each test. Or did I miss something?